platform/darwin/src/MGLLoggingConfiguration_Private.h

```diff
@@ -4,6 +4,13 @@ NS_INLINE NSString *MGLStringFromBOOL(BOOL value) {
     return value ? @"YES" : @"NO";
 }
 
+#if TARGET_OS_IPHONE
+#import <UIKit/UIKit.h>
+NS_INLINE NSString *MGLStringFromSize(CGSize size) { return NSStringFromCGSize(size); }
```

Comment:
This introduces a dependency on UIKit in libmbgl-filesource, which is unfortunate. I think we should try to limit the dependency to Core Graphics, which defines `CGSize`. I’m pretty sure `NSStringFromSize()` and `NSStringFromCGSize()` just serialize the result of `CGSizeCreateDictionaryRepresentation()`, which is also defined in CGGeometry.h.

Reply:
I’ll do some digging. 🤔

Reply:
Moving this definition to a header that already has UIKit and/or CoreGraphics would be the simplest solution — `MGLGeometry_Private.h` looks like the best candidate.

Reply:
→ 1636ed7257945a33b40f61789c5def220e78c3b3